src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -5488,16 +5488,235 @@ static void Main()
             string expectedOutput = "bb";
             var compilation = CreateCompilation(source, options: TestOptions.ReleaseExe, parseOptions: TestOptions.RegularWithPatternCombinators);
             compilation.VerifyDiagnostics(
-                // (7,21): warning CS8509: The switch expression does not handle all possible values of its input type (it is not exhaustive).
-                //         var str = x switch // does not handle zero
-                Diagnostic(ErrorCode.WRN_SwitchExpressionNotExhaustive, "switch").WithLocation(7, 21),
-                // (15,17): warning CS8509: The switch expression does not handle all possible values of its input type (it is not exhaustive).
-                //         str = x switch // does not handle zero
-                Diagnostic(ErrorCode.WRN_SwitchExpressionNotExhaustive, "switch").WithLocation(15, 17)
+                    // (7,21): warning CS8509: The switch expression does not handle all possible values of its input type (it is not exhaustive). For example, the pattern '0' is not covered.
+                    //         var str = x switch // does not handle zero
+                    Diagnostic(ErrorCode.WRN_SwitchExpressionNotExhaustive, "switch").WithArguments("0").WithLocation(7, 21),
+                    // (15,17): warning CS8509: The switch expression does not handle all possible values of its input type (it is not exhaustive). For example, the pattern '0' is not covered.
+                    //         str = x switch // does not handle zero
+                    Diagnostic(ErrorCode.WRN_SwitchExpressionNotExhaustive, "switch").WithArguments("0").WithLocation(15, 17)
                 );
             var compVerifier = CompileAndVerify(compilation, expectedOutput: expectedOutput);
         }
 
+        [Fact]
+        public void NonexhaustiveEnumDiagnostic_01()
+        {
+            var source =
+@"
+class C
+{
+    int M(Color color) => color switch
+    {
+        Color.Red => 0,
+        Color.Blue => 2,
+    };
+}
+enum Color { Red, Greed, Blue }
```

Comment:
>Greed [](start = 18, length = 5)

Typo? Here and in tests below?

Reply:
No, it is a kind of running joke. Greed is the color of money.

---
In reply to: [443141501](https://github.com/dotnet/roslyn/pull/44702#discussion_r443141501) [](ancestors = 443141501)